index.js

```diff
@@ -118,6 +114,7 @@ module.exports = {
     'scss/at-else-empty-line-before': 'never',
     'scss/at-if-closing-brace-newline-after': 'always-last-in-chain',
     'scss/at-if-closing-brace-space-after': 'always-intermediate',
+    'scss/at-import-partial-extension': 'always',
```

Comment:
As noted in the migration guide:

> `stylelint-scss` `v5` introduced the `scss/at-import-partial-extension` rule. This was initially set to `never`. This does not work for us as it throws false positives on common paths like `@skyscanner/bpk-foundations-web/tokens/base.default.scss`, when the `.scss` is removed it fails on .`default` and is not configurable. Because of this we override to `always`.